lib/recipes-data/src/lib/s3.ts

```diff
@@ -59,7 +69,7 @@ export async function removeRecipeContent(recipeSHA: string, purgeType?:"soft"|"
   const realAttempt = attempt ?? 1;
 
   const Key = `content/${recipeSHA}`;
-  console.debug(`DEBUG: removeRecipeContent path is s3://${Bucket}/${Key}`);
+  console.debug(`DEBUG: removeRecipeContent path is s3://${Bucket ?? "(no bucket)"}/${Key}`);
```

Comment:
If it's possible that `Bucket` doesn't exist, we should check that and abort this entire feature if it's missing